xoinvader/tests/test_enemy_wave.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+
+from xoinvader.enemy_wave import EnemyWave
+
+
+def test_wave():
+    e = EnemyWave()
+
+    a = 0
+    b = 0
+    def inc_a():
```

Comment:
I disagree.

I test for callbacks firing at proper times here, and this is the simplest possible closure example that I managed to write. Adding objects is an unnecessary complication here.

All thanks go to Python lambda assignment retardation.